src/beautifiers/eslint.coffee

```diff
@@ -14,21 +15,35 @@ module.exports = class ESLintFixer extends Beautifier
   }
 
   beautify: (text, language, options) ->
-    return new @Promise((resolve, reject) ->
+    return new @Promise((resolve, reject) =>
       editor = atom.workspace.getActiveTextEditor()
       filePath = editor.getPath()
       projectPath = atom.project.relativizePath(filePath)[0]
 
       result = null
-      allowUnsafeNewFunction ->
+      allowUnsafeNewFunction =>
         importPath = Path.join(projectPath, 'node_modules', 'eslint')
+        eslint = null
         try
-          CLIEngine = require(importPath).CLIEngine
-
-          cli = new CLIEngine(fix: true, cwd: projectPath)
-          result = cli.executeOnText(text).results[0]
-
-          resolve result.output
-        catch err
-          reject(err)
+          eslint = requireg('eslint', true)
+        catch
+          try
+            eslint = require(importPath)
+          catch
+            try
+              eslint = require('eslint')
+            catch
+              needInstallMsg = 'Error: it seems like you did not install eslint globally,'
```

Comment:
The current recommended way to install ESLint is not globally: https://eslint.org/docs/user-guide/getting-started#installation-and-usage

This leads me to the conclusion the order of line 28 to line 34 should be changed:

1) `require(importPath` - Project specific
2) `requireg('eslint', true)` - Globally
3) `require('eslint')` - Atom-Beautify

[Since Atom-Beautify has it installed as a direct dependency](https://github.com/Glavin001/atom-beautify/blob/1498555b42591bbdcc1488b1ec02e9d787da0cc3/package.json#L188), there should always be at least one installation of ESLint found and this error message should never be reached.